### PR TITLE
New Header update w/ clickable userid

### PR DESCRIPTION
**Pull request by minybrandon**
header components correctly aligned using flex

image

### CODE REVIEW

File: src/posts/PublishedDate.js [hunk 1]
```diff
@@ -3,7 +3,10 @@ import React from 'react';
 class PublishedDate extends React.Component{
     render(){
         return(
-            <p>{this.props.publishedDate.toString()}</p>
+            <div className="left-align-board">
```
Comment (justinyum98) [inline src/posts/PublishedDate.js]: `className="left-align-board"` by itself won't left align the board. You will need to use CSS, or Material-UI's [styling feature](https://material-ui.com/styles/basics/). Give this a read.

File: src/posts/UserId.js [hunk 2]
```diff
@@ -3,9 +3,13 @@ import React from 'react';
 class UserId extends React.Component{
     render(){
         return(
-            <p>{this.props.userId}</p>
+            <button onClick={() => console.log(this.props.userId)}>
+                {this.props.userId}
+            </button>
```
Comment (justinyum98) [inline src/posts/UserId.js]: Nice 👍

File: src/posts/UserId.js [hunk 3]
```diff
@@ -3,9 +3,13 @@ import React from 'react';
...
         )
     }
 }
 
+
+
```
Comment (justinyum98) [inline src/posts/UserId.js]: Delete these extra lines, no purpose in them.

**Review by justinyum98 — CHANGES_REQUESTED** [review]
Suggested some changes 👍

File: src/posts/PublishedDate.js [hunk 4]
```diff
@@ -3,7 +3,10 @@ import React from 'react';
...
+                <p>{this.props.publishedDate.toString()}</p>
```
Comment (sangyum) [inline src/posts/PublishedDate.js]: `publishedDate` is a Javascript Date object. Javascript Date object has many function that prints out the date in different format. `toString()` is probably not the function we want to use. Consider other functions, after reviewing https://www.w3schools.com/jsref/jsref_todatestring.asp and other functions.

File: src/posts/UserId.js (hunk 2 above)
Comment (sangyum) [reply src/posts/UserId.js]: ➕

File: src/stories/3-Header.stories.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+};
+
+const publishedDate = new Date();
+
+const userId = 'byum1996'
+
+
+export const publishedRender = () => <PublishedDate publishedDate={publishedDate} />
+export const userIdRender = () => <UserId userId={userId} />
+export const headerRender = () => <Header publishedDate={publishedDate} userId={userId}/>
```
Comment (sangyum) [inline src/stories/3-Header.stories.js]: line feed at the end of file

**Review by justinyum98 — APPROVED** [review]
👍

**Pull request merged**